content/developers/office_hours.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Community Office Hours
+type: documentation
+---
+
+## Community Code Review
+
+Datadog runs twice monthly code reviews and office hours to assist community members who are contributing to our [open source projects](#list-of-open-source-project).  
+
+**These sessions are held on the 2nd and 4th Wednesday of each month.**
+
+### Where
+
+Office hours will be held in #integrations in the [Datadog Community Slack][1] ([get an invitation][2]), in conjunction with a [Google Hangout][3].
+
+### When
+
+* 2nd Wednesday of each month at 10:00am eastern time (2:00pm UTC).  
```

Comment:
Thanks to the magic of daylight savings time, these correlations are not always true (EST is UTC-5, EDT is UTC-4). Since we aren't scheduling this according to UTC, the only safe way to express the schedule is:
> * 2nd Wednesday of each month at 10:00 AM Eastern Time.
> * 4th Wednesday of each month at 3:00 PM Eastern Time.